cli/cmd/vuln_container.go

```diff
@@ -199,20 +206,21 @@ filter on containers with vulnerabilities that have fixes available.`,
 				return cli.OutputJSON(filteredAssessments)
 			}
 
+			// Build data
 			rows := vulAssessmentsToTable(filteredAssessments)
+			headers := []string{"Registry", "Repository", "Last Scan", "Status", "Containers", "Vulnerabilities", "Image Digest"}
 
+			switch {
```

Comment:
maaaann!!! I have never seen this switch statement, I LOVE IT! 🥇 

![tenor-120575326](https://user-images.githubusercontent.com/5712253/148783523-009c1f61-cc71-4e9b-a988-72a84d7e82df.gif)

Reply:
🙌